README.md

```diff
@@ -1 +1,5 @@
-# trinug
\ No newline at end of file
+# trinug
+
+yay trinug
+
+test 2
```

Comment:
Um, I think this is meant to be test 3.

/cc @ghtstr

Reply:
No, I think it should be 2.